AEPMedia/Sources/MediaCollection/MediaCollectionReportHelper.swift

```diff
@@ -41,19 +43,39 @@ class MediaCollectionReportHelper {
     /// - host: Host component of the URL
     /// - sessionId: the session id of the Media session
     /// - Returns: URL for sending media hits
-    static func getTrackingURLForEvents(host: String, sessionId: String?) -> URL? {
-        guard !host.isEmpty, let sessionId = sessionId, !sessionId.isEmpty else {
+    static func getTrackingURLForEvents(trackingServer: String, sessionId: String?) -> URL? {
+        guard !trackingServer.isEmpty, let sessionId = sessionId, !sessionId.isEmpty else {
             Log.warning(label: LOG_TAG, "[\(Self.CLASS_NAME)<\(#function)>] - Unable to create tracking url for events. Host (MediaCollectionServer) url or collection server sessionId is empty.")
             return nil
         }
 
+        let serverAddress = getHostAndPath(trackingServer)
+
         var urlComponents = URLComponents()
         urlComponents.scheme = "https"
-        urlComponents.host = host
-        urlComponents.path = "/api/v1/sessions/\(sessionId)/events"
+        urlComponents.host = serverAddress["host"]
+        urlComponents.path = (serverAddress["path"] ?? "") + "/api/v1/sessions/\(sessionId)/events"
         return urlComponents.url
     }
 
+    static func getHostAndPath(_ trackingServer: String) -> [String: String] {
+        var ret =  [String: String]()
+        let trackingServerComponents = trackingServer.components(separatedBy: "/")
```

Comment:
Is there any chance here that the `trackingServer` contains a scheme, such as `https://` such that separating the string by `/` will fail to set the host in array index 0?